helm/kubernetes-cluster-autoscaler-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    giantswarm.io/service-type: "{{ .Values.serviceType }}"
+    k8s-app: {{ .Values.name }}
```

Comment:
AFAIK this is not k8s-app, so the label should be just `app`

k8s-app is component like `calico` or `kube-proxy` which is actually doing some changes in k8s